src/graphx/graphx.asm

```diff
@@ -133,6 +133,13 @@ library 'GRAPHX', 11
 ; v10 functions
 ;-------------------------------------------------------------------------------
 	export gfx_CopyRectangle
+;-------------------------------------------------------------------------------
+; v11 functions
```

Comment:
Should this be v12 or are the comments off by a version?

Reply:
I had a look through the history and it appears that version 10 added no functions. So yes, this should be v12. And the section above should be v11.